rfcs/0001-bootstrapping-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: bootstrapping-rfc-process
```

Comment:
This RFC is missing `Current Status` and `Owner` headers

Reply:
@flyingzumwalt seems the other RFCs you've done are also missing this, did you use `0000-template-rfc.md ` as a template for these?

Reply:
@VictorBjelkholm they were created before the template was updated.

Reply:
✅  added. yes. those fields were added to the template after these PRs were created